client/components/ProjectSuitableBuddy.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { Feather } from '@expo/vector-icons';
+
+
+const ProjectSuitableBuddy = () => {
+
+    const BuddyView = styled.View`
+    display: flex;
+    flex-direction: row;
+    margin-top: 7px;
+    `
+    
+    const BuddyText = styled.Text`
+    margin-left: 13px;
+
```

Comment:
styled components should be outside of the component definition as they need to be static